### ISSUE TITLE
Why could not load qgis_app.dll?

### ISSUE THREAD

**Issue by memorylorry** [qgis/QGIS#36826]
**Describe the bug**

I have built QGIS final-3.10.4 and met a problem like this.
``` code
Details:
Could not load qgis_app.dll
Windows Error: 找不到指定的模块。
Help:
Check E:/QGIS3104/ms-windows/osgeo41/build-qgis-test-x86-64/output/bin/RelWithDebInfo/qgis.env for correct environment paths
```
As for this issue, I tried to fix its dependecies but I cannot find all `dll` needed. There are 90 dll files left which I have not found. 

<details>

<summary>Missing Dependencies!!!</summary>

API-MS-WIN-APPMODEL-STATE-L1-2-0.DLL

API-MS-WIN-CORE-APIQUERY-L1-1-1.DLL

API-MS-WIN-CORE-COM-L1-1-1.DLL

API-MS-WIN-CORE-COM-L1-1-2.DLL

API-MS-WIN-CORE-COM-MIDLPROXYSTUB-L1-1-0.DLL

API-MS-WIN-CORE-COM-PRIVATE-L1-1-1.DLL

API-MS-WIN-CORE-COM-PRIVATE-L1-2-0.DLL

API-MS-WIN-CORE-CONSOLE-L1-2-1.DLL

API-MS-WIN-CORE-CONSOLE-L2-2-0.DLL

API-MS-WIN-CORE-CONSOLE-L3-2-0.DLL

API-MS-WIN-CORE-DATETIME-L1-1-2.DLL

API-MS-WIN-CORE-ERRORHANDLING-L1-1-2.DLL

API-MS-WIN-CORE-ERRORHANDLING-L1-1-3.DLL

API-MS-WIN-CORE-FIBERS-L2-1-0.DLL

API-MS-WIN-CORE-FILE-L1-2-2.DLL

API-MS-WIN-CORE-FILE-L2-1-2.DLL

API-MS-WIN-CORE-FILE-L2-1-3.DLL

API-MS-WIN-CORE-HEAP-L2-1-0.DLL

API-MS-WIN-CORE-JOB-L1-1-0.DLL

API-MS-WIN-CORE-JOB-L2-1-0.DLL

API-MS-WIN-CORE-KERNEL32-PRIVATE-L1-1-2.DLL

API-MS-WIN-CORE-LARGEINTEGER-L1-1-0.DLL

API-MS-WIN-CORE-LIBRARYLOADER-L1-2-1.DLL

API-MS-WIN-CORE-LIBRARYLOADER-L1-2-2.DLL

API-MS-WIN-CORE-LIBRARYLOADER-L2-1-0.DLL

API-MS-WIN-CORE-LOCALIZATION-L1-2-2.DLL

API-MS-WIN-CORE-LOCALIZATION-L2-1-0.DLL

API-MS-WIN-CORE-LOCALIZATION-PRIVATE-L1-1-0.DLL

API-MS-WIN-CORE-MEMORY-L1-1-3.DLL

API-MS-WIN-CORE-NAMEDPIPE-L1-2-1.DLL

API-MS-WIN-CORE-NAMEDPIPE-L1-2-2.DLL

API-MS-WIN-CORE-NAMESPACE-L1-1-0.DLL

API-MS-WIN-CORE-NORMALIZATION-L1-1-0.DLL

API-MS-WIN-CORE-PATH-L1-1-0.DLL

API-MS-WIN-CORE-PERFCOUNTERS-L1-1-0.DLL

API-MS-WIN-CORE-PROCESSSNAPSHOT-L1-1-0.DLL

API-MS-WIN-CORE-PROCESSTHREADS-L1-1-3.DLL

API-MS-WIN-CORE-PROCESSTOPOLOGY-L1-1-0.DLL

API-MS-WIN-CORE-PSAPI-ANSI-L1-1-0.DLL

API-MS-WIN-CORE-PSAPI-L1-1-0.DLL

API-MS-WIN-CORE-PSAPI-OBSOLETE-L1-1-0.DLL

API-MS-WIN-CORE-PSM-KEY-L1-1-0.DLL

API-MS-WIN-CORE-QUIRKS-L1-1-0.DLL

API-MS-WIN-CORE-REGISTRY-L1-1-1.DLL

API-MS-WIN-CORE-REGISTRY-L1-1-2.DLL

API-MS-WIN-CORE-REGISTRYUSERSPECIFIC-L1-1-0.DLL

API-MS-WIN-CORE-SIDEBYSIDE-L1-1-0.DLL

API-MS-WIN-CORE-STRING-L2-1-1.DLL

API-MS-WIN-CORE-SYNCH-L1-2-1.DLL

API-MS-WIN-CORE-SYSINFO-L1-2-3.DLL

API-MS-WIN-CORE-SYSTEMTOPOLOGY-L1-1-0.DLL

API-MS-WIN-CORE-SYSTEMTOPOLOGY-L1-1-1.DLL

API-MS-WIN-CORE-VERSION-L1-1-1.DLL

API-MS-WIN-CORE-VERSION-PRIVATE-L1-1-0.DLL

API-MS-WIN-CORE-VERSIONANSI-L1-1-0.DLL

API-MS-WIN-CORE-VERSIONANSI-L1-1-1.DLL

API-MS-WIN-CORE-WINDOWSERRORREPORTING-L1-1-0.DLL

API-MS-WIN-CORE-WINDOWSERRORREPORTING-L1-1-1.DLL

API-MS-WIN-CORE-WINDOWSERRORREPORTING-L1-1-2.DLL

API-MS-WIN-CORE-WINRT-ERROR-L1-1-0.DLL

API-MS-WIN-CORE-WINRT-ERROR-L1-1-1.DLL

API-MS-WIN-CORE-WOW64-L1-1-1.DLL

API-MS-WIN-CORE-XSTATE-L2-1-1.DLL

API-MS-WIN-EVENTING-CONSUMER-L1-1-1.DLL

API-MS-WIN-GDI-DPIINFO-L1-1-0.DLL

API-MS-WIN-GDI-INTERNAL-UAP-L1-1-0.DLL

API-MS-WIN-MM-MISC-L1-1-0.DLL

API-MS-WIN-MM-MME-L1-1-0.DLL

API-MS-WIN-MM-TIME-L1-1-0.DLL

API-MS-WIN-SECURITY-ACTIVEDIRECTORYCLIENT-L1-1-0.DLL

API-MS-WIN-SECURITY-APPCONTAINER-L1-1-0.DLL

API-MS-WIN-SECURITY-AUDIT-L1-1-0.DLL

API-MS-WIN-SECURITY-AUDIT-L1-1-1.DLL

API-MS-WIN-SECURITY-BASE-PRIVATE-L1-1-0.DLL

API-MS-WIN-SECURITY-CAPABILITY-L1-1-0.DLL

API-MS-WIN-SECURITY-GROUPPOLICY-L1-1-0.DLL

API-MS-WIN-SECURITY-LSALOOKUP-L1-1-2.DLL

API-MS-WIN-SERVICE-CORE-L1-1-2.DLL

API-MS-WIN-SERVICE-PRIVATE-L1-1-2.DLL

API-MS-WIN-SERVICE-PRIVATE-L1-1-3.DLL

API-MS-WIN-SERVICE-PRIVATE-L1-1-4.DLL

API-MS-WIN-SHCORE-PATH-L1-1-0.DLL

API-MS-WIN-SHCORE-SCALING-L1-1-1.DLL

API-MS-WIN-SHCORE-SYSINFO-L1-1-0.DLL

API-MS-WIN-SHCORE-TASKPOOL-L1-1-0.DLL

API-MS-WIN-SHELL-SHELLCOM-L1-1-0.DLL

API-MS-WIN-SHELL-SHELLFOLDERS-L1-1-0.DLL

API-MS-WIN-STATESEPARATION-HELPERS-L1-1-0.DLL

API-MS-WIN-STORAGE-EXPORTS-EXTERNAL-L1-1-0.DLL

API-MS-WIN-STORAGE-EXPORTS-INTERNAL-L1-1-0.DLL

</details>

Hope anyone who have met it can help me.

**How to Reproduce**

Refer to the guidance provided by [this repository](https://github.com/qgis/QGIS/blob/final-3_10_4/INSTALL).

**QGIS and OS versions**

QGIS VER : final-3.10.4

OS VER : Windows 10 Home Edition

Visual Studio VER : 2017 Professional

**Comment by gioman** [comment]
@memorylorry I think that this would be more appropriate for a question in the developers mailing list rather than considering this as a bug.

**Comment by memorylorry** [comment]
Thanks for your notifications and I close this issue.

**Comment by cnkant** [comment]
I encountered the same issue when I tried to run QGIS versions 3.16, 3.22, and 3.28. Even after reinstalling the program using OSGeo4W, I kept getting an error message saying that OpenCL.dll was missing. Before encountering this problem, I had removed my NVIDIA graphics card from the device and uninstalled its drivers. So, I suspected that the issue might be related to the graphics card driver. Currently, my computer is using the integrated graphics card of an Intel CPU. I reinstalled its driver, and the problem was resolved.
